pkg/operation/cloudbotanist/alicloudbotanist/controlplane.go

```diff
@@ -135,6 +136,12 @@ func (b *AlicloudBotanist) GenerateCSIConfig() (map[string]interface{}, error) {
 	)
 }
 
+// MetadataServiceAddress returns Aliyun's MetadataService address.
+func (b *AlicloudBotanist) MetadataServiceAddress() (*net.IPNet, error) {
```

Comment:
Do we even need this function? you are statically passing the IPNet output yourself as a string, what's the value, if I provide `100.100.100.200/32` I get back `100.100.100.200/32`, MetadataService IPs are always `/32` maybe it would have been useful in different cases where you are not always using a `/32`.